src/drive/web/modules/queries.js

```diff
@@ -28,7 +28,12 @@ export const formatFolderQueryId = (
   return `${type} ${folderId} ${sortAttribute} ${sortOrder}`
 }
 
-const buildQuery = ({ currentFolderId, type, sortAttribute, sortOrder }) => ({
+const buildDriveQuery = ({
```

Comment:
I do not find it that much clearer, but it sure seems that buildQuery is not specific enough. I suggest `buildFolderContentQuery` ?

Reply:
I want to rename `DriveView` to `FolderView` and rename this query to `FolderQuery`. 

I'm currently fixing a few bugs in FilesViewer, I'll do it in a later commit